system/jlib/jlog.hpp

```diff
@@ -756,6 +756,11 @@ extern jlib_decl const LogMsgJobInfo unknownJob;
 
 extern jlib_decl ILogMsgManager * queryLogMsgManager();
 extern jlib_decl ILogMsgHandler * queryStderrLogMsgHandler();
+extern jlib_decl void updateContainerizedLogMsgHandlerDetail(LogMsgDetail logDetail);
+extern jlib_decl void updateContainerizedLogMsgHandlerLogFields(unsigned fields);
+extern jlib_decl void updateContainerizedLogMsgHandler(ILogMsgFilter * newFilter);
+extern jlib_decl void updateContainerizedLogMsgHandler(IPropertyTree * tree);
+extern jlib_decl void updateContainerizedLogMsgHandlerFromConfig();
```

Comment:
I don't think need all these. It should get all from config and component defaults.
So a single installLogHandlerFromConfig() should be good enough. It is an error if a component config doesn't exist (i.e. queryComponentConfig() will throw an exception).